lib/cocoapods/downloader/cache.rb

```diff
@@ -12,6 +12,8 @@ class Cache
       #
       attr_reader :root
 
+      # Init a new instance
```

Comment:
Considering you used `Initialize a new instance` elsewhere, I'd op to use initialise here too for consistency and full words.